src/pages/library/DataBrowserDetails.js

```diff
@@ -0,0 +1,82 @@
+import { Fragment, useState } from 'react'
+import { div, h, h1 } from 'react-hyperscript-helpers'
+import FooterWrapper from 'src/components/FooterWrapper'
+import { centeredSpinner, icon } from 'src/components/icons'
+import { libraryTopMatter } from 'src/components/library-common'
+import * as Utils from 'src/libs/utils'
+
+
+const activeTab = 'browse & explore'
+const styles = {
+  content: { padding: 20, marginTop: 15 }
+}
+
+const getSnapshot = id => new Promise(resolve => setTimeout(() => {
+  resolve({
+    namespace: 'harry-potter',
+    name: 'Harry Potter',
+    created: '2020-01-07T18:25:28.340Z',
+    lastUpdated: '2020-01-07T18:25:28.340Z',
+    tags: {
+      itemsType: 'AttributeValue',
+      items: ['1000 Genomes', 'CMG', 'Open Access', 'Exome']
+    },
+    description: 'The boy who lived',
+    lowerName: 'harry potter',
+    lowerDescription: 'lower description',
+    project: {
+      id: '112',
+      name: 'CDC'
+    },
+    subjects: 10,
+    dataType: 'RNA Seq',
+    keepCollapsed: true,
+    locked: false,
+    files: 15,
+    fileSize: 4432
+  })
+}, 2000))
+
+const MainContent = snapshot => {
```

Comment:
This needs to be destructured to pull the snapshot out
```suggestion
const MainContent = ({ snapshot }) => {
```